regulations/views/reader.py

```diff
@@ -27,22 +27,15 @@ def get_context_data(self, **kwargs):
         # answering the question: what are we looking at?
         reg_version = context["version"]
         reg_part = context["part"]
-        reg_citation = context["citation"]
-        toc = self.get_toc(reg_part, reg_version)
-        meta = utils.regulation_meta(reg_part, reg_version)
-        tree = self.get_regulation(reg_citation, reg_version)
-
-        if not meta:
-            raise Http404
+        tree = self.client.v2_part(reg_version, 42, reg_part)
 
         c = {
-            'tree':         tree,
+            'tree':         tree['document'],
             'reg_part':     reg_part,
-            'meta':         meta,
-            'TOC':          toc,
+            'structure':    tree['structure']['children'][0]['children'][0]['children'][0],
```

Comment:
This could error, but I suppose that would be fine since it'll just throw 500 anyway

Reply:
definitely something that could be improved